readme.md

```diff
@@ -112,6 +112,12 @@ Returns a ("live") pull stream that emits the message key (string) for thread ro
 * `opts.allowlist`: optional array of strings. Dictates which messages **types** to allow as root messages, while forbidding other types.
 * `opts.blocklist`: optional array of strings. Dictates which messages **types** to forbid as root messages, while allowing other types.
 
+### `ssb.threads.hashtagsMatching(opts)`
```

Comment:
The API seems to be `(opts, cb)`, so this needs to change. Also the sentence says "returns a pull stream", should say "calls the callback with...". :)

Reply:
addressed via 43d2270